src/main/java/org/squiddev/plethora/integration/vanilla/method/MethodsKineticEntity.java

```diff
@@ -54,6 +54,9 @@ public static void look(@FromSubtarget EntityLivingBase target, double yaw, doub
 		yaw %= 360;
 		pitch %= 360;
 
+		// Clamp pitch between -90 and 90 degrees for aesthetics.
+		pitch = pitch > 90 ? 90 : pitch < -90 ? -90 : pitch;
```

Comment:
I think there's a `MathHelper.clamp` method, which may be a wee bit nicer.

Reply:
Oh good, because I hate ternary operators.

Reply:
Resolved with https://github.com/SquidDev-CC/plethora/pull/218/commits/5a92e625f609a82fe804c0dea58fba93aed9d65e